docs/fides/docs/language/resources/policy.md

```diff
@@ -9,9 +10,9 @@ A Policy is your privacy policy as code, it lists a set of acceptable and non-ac
 
 ## Object Structure
 
-**fides_key**<span class="required"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_string_
+**fides_key**<span class="required"/>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;_constrained string_
 
-A string token of your own invention that uniquely identifies this Policy. It's your responsibility to ensure that the value is unique across all of your Policy objects. The value may only contain alphanumeric characters and underbars (`[A-Za-z0-9_]`).
+A string token of your own invention that uniquely identifies this Policy. It's your responsibility to ensure that the value is unique across all of your Policy objects. The value may only contain alphanumeric characters and underbars (`[A-Za-z0-9_.-]`).
```

Comment:
```suggestion
A custom string token that uniquely identifies this Policy. It must be unique across all Policy objects. The value may only contain alphanumeric characters, `_`, `.`, and `-` (it will be evaluated against the regular expression: `[A-z0-9_.-]`).
```